class/practice3.1/src/testRepeater.cpp

```diff
@@ -0,0 +1,60 @@
+#include <print>
+#include <Helpers.hpp>
+#include <UnitTests.hpp>
+#include <string>
+#include <vector>
+int main() {
+    UnitTests testSuite;
+
+    testSuite.addTest("Repeater_4homes",
+    [](){
+        std::string res="home";
```

Comment:
test cases seems to be good.
Minor:
1. add better formatting - even empty lines between blocks Build/Operate/Check (or Arrange/Act/Assert if you like this abbreviation more)
2. 2. add spaces to have consistent codestyle like  StringReapeater(vec, res); instead of StringReapeater(vec,res);